src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
replace 100 with constant 
```suggestion
        int number = new Random().nextInt(100);
```